cmd/lp-colouring/colouring.go

```diff
@@ -78,10 +78,8 @@ func (*Colouring) BaseVertexMailbox(vertex *graph.Vertex[VertexProperty, EdgePro
 	edgeAmount := len(vertex.OutEdges)        // Note: will be zero for dynamic graphs.
 	m.NbrScratch = make([]uint32, edgeAmount) // This is for a thread unsafe view // m.NbrScratch.Store(ns)
 
-	if edgeAmount > 0 {
-		for i := 0; i < edgeAmount; i++ {
-			m.NbrScratch[i] = EMPTY_VAL
-		}
+	for i := 0; i < edgeAmount; i++ {
```

Comment:
```suggestion
	for i := 0; i < edgeAmount; i++ { // Note: does nothing for dynamic graphs.
```

🙃 I think I did that to help differentiate from dynamic, but a comment is better!